Add paid shared nodes (#333)
chainstack/docs: docs/operations/bitcoin/types.md

Chainstack supports deploying a Bitcoin node of the following types:

> * Shared — the node resources are shared with other platform users. With a shared node, you pay for JSON-RPC requests to the node and do not pay for the compute and storage resources used by the node.

Reviewer: ```suggestion
* Shared — the node resources are shared with other platform users. With a shared node, you only pay for JSON-RPC requests to the node.
```

Author: same

> * Dedicated — the node resources are exclusive to you. With a dedicated node, you pay for the compute and storage resources used by the node and do not pay for JSON-RPC requests to the node.

Reviewer: ```suggestion
* Dedicated — the node resources are exclusive to you. With a dedicated node, you pay for the compute and storage resources used by the node. The JSON-RPC requests to dedicated nodes are free of charge.
```

Author: same. requests being free of charge is not the explanatory point here and in other cases in this pull request. it's not free in the end, it's the user not specifically paying for requests in this specific setup.
